### ISSUE TITLE
add SSID "franken.freifunk.net"

### ISSUE THREAD

**Issue by mojoaxel** [WIStudent/FreifunkAutoConnectApp#13]
thx :-)

**Comment by WIStudent** [comment]
"franken.freifunk.net" is already included in the current list of SSIDs ( https://github.com/WIStudent/freifunk-ssids ). Are you sure that you are using the latest version? Since version 0.5 the app should automatically check for a new verison of the list when it is opened. (If the last successful check was in the last 24 hours, the app will not check for a new SSID list until 24h have passed since the last check.) There was also a bug that prevented showing new SSIDs right after they were downloaded if the SSID filter was used. A workaround was to close the app in the Android settings and then restarting it. The bug got fixed in version 0.6. Unfortunately version 0.6 and up are not availiable on FDroid, they didn't like that I included the Google Play services location API in version 0.6 although I replaced it in 0.6.1 with an open source alternative.
